### ISSUE TITLE
Difference in output brightness based on rotation

### ISSUE THREAD

**Issue by jovan2009** [Beep6581/RawTherapee#5145]
Hi, I was editing a jpeg file and found out that when I tilt the image the output image has wrong colors (too dark). I attached in the zip file original jpeg, the output files and settings used. The "untilted" has correct colors, as soon as I tilt it to correct for slope the output is wrong. While editing image looks correct, only the output image is too dark.
[Rawtherapee bug report.zip](https://github.com/Beep6581/RawTherapee/files/2763878/Rawtherapee.bug.report.zip)

**Comment by heckflosse** [comment]
Can you try without tonemapping please?

**Comment by jovan2009** [comment]
Yes, I tried without tonemapping and it seems tilted and not tilted outputs have the same colors.

**Comment by Beep6581** [comment]
Tested on `5.5-91-g`f84a75de1

Using `amsterdam.pef` and `neutral`, I narrowed this down to the following tools with sample values:
* Exposure > Clip out-of-gamut colors: disabled.
* Exposure > Highlight compression: 500
* Exposure > Black: -16384
* Exposure > Shadow compression: 100
* Tone Mapping: enabled, defaults.
* Rotate: -7 (exact value doesn't really matter, the important thing is probably to push some clipped area out of frame).

![amsterdam_5145_rotate_off](https://user-images.githubusercontent.com/5844619/51865456-fc5ec000-234e-11e9-978a-357c389b61f5.jpg)
[amsterdam_5145_rotate_off.jpg.out.pp3.txt](https://github.com/Beep6581/RawTherapee/files/2804884/amsterdam_5145_rotate_off.jpg.out.pp3.txt)
![amsterdam_5145_rotate_on](https://user-images.githubusercontent.com/5844619/51865462-02ed3780-234f-11e9-9bc1-c105b31904a7.jpg)
[amsterdam_5145_rotate_on.jpg.out.pp3.txt](https://github.com/Beep6581/RawTherapee/files/2804885/amsterdam_5145_rotate_on.jpg.out.pp3.txt)

**Comment by Lawrence37** [comment]
Cannot reproduce on `dev` using amsterdam_5145_rotate_on.jpg.out.pp3.txt:
![amsterdam](https://github.com/Beep6581/RawTherapee/assets/45837045/cf8e66b2-809e-4557-b8d1-7d57c23b56ac)
